kfs-core/src/main/java/edu/arizona/kfs/pdp/businessobject/ShippingHeader.java

```diff
@@ -57,7 +68,20 @@ public void setOpenCustomField(String masterEDINumber) {
 		
     	map.put("creationDate", getCreationDate().toString());
     	map.put("transactionRefNumber", getTransactionRefNumber());
-        return map;
+    	return map;
+    }
+
+    public List<ShippingInvoice> getInvoices() {
+        if ( !invoicesLoaded ) {

```

Comment:
Not sure if we need a boolean for invoicesLoaded, if invoices is null or isEmpty() can suffice.